cppwinrt/component_writers.h

```diff
@@ -868,7 +869,7 @@ catch (...) { return winrt::to_hresult(); }
 namespace winrt::@::implementation
 {
     template <typename D, typename... I>
-    using %T = %_base<D, I...>;
+    using %T = %_base<D, impl::marker, I...>;
```

Comment:
for classes without interfaces, this produced a compile error - not enough template parameters.  supplying the marker is a simple way to resolve the argument count.